deploy/backup_tarantool.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
+
+# Remote server information
+SERVER_USER="root"
+SERVER_HOST="benalu.dev"
+SSH_PORT=22
+
+# Tarantool information
+TT_USER="userT"
+TT_PASS="passT"
+TT_PORT=3301
+TT_PORT_LOCAL=3302
+TT_DATA_DIR=/home/$SERVER_USER/tmpdb/var_lib_tarantool
+CONN_STR=$TT_USER:$TT_PASS@localhost:$TT_PORT_LOCAL
+
+BACKUP_DATE=$(date '+%Y%m%d_%H%M%S')
+BACKUP_DIR=./backup/tt_${BACKUP_DATE}
+
+mkdir -p $BACKUP_DIR
+ssh -N -f -L $TT_PORT_LOCAL:localhost:$TT_PORT $SERVER_USER@$SERVER_HOST
```

Comment:
i think this one should be `ssh .... 'the command you want to run on server side'`

Reply:
nevermind, i got what it means, create a tunnel forever